src/components/case_manager/StudentParaForm.tsx

```diff
@@ -0,0 +1,45 @@
+/* This is a functional <form> component to be imported by both the AllStudents.tsx and the AllParas.tsx
+NOTE: Subsequent list rendering and trpc calls would be challenging to reuse, but this form element can be used successfully in the front end for users to create or add students or paras to their caseloads, populating both lists. */
+
+import React from "react";
+import styles from "../../styles/Dashboard.module.css";
+
+interface Props {
+  title: string;
+  handleSubmit: (event: React.FormEvent<HTMLFormElement>) => void;
```

Comment:
the convention in React is to usually have event handler in props be named something like `on...`, so maybe

```suggestion
  onSubmit: (event: React.FormEvent<HTMLFormElement>) => void;
```

?